src/modules/FairFindPackage2.cmake.in

```diff
@@ -39,6 +39,62 @@ if(@PROJECT_NAME@_FOUND)
   list(APPEND PROJECT_PACKAGE_DEPENDENCIES @PROJECT_NAME@)
 endif()
 
+# private helper function to find least common denominator in given
+# true-thy strings
```

Comment:
Maybe rename to lcp? longest common prefix?

Reply:
done